distribution.md

```diff
@@ -37,35 +37,82 @@ A `.cnb` file MUST be an uncompressed tar archive containing an OCI image. Its f
 
 [‡](README.md#windows-only)For Windows buildpackages, all FS layers MUST be either buildpack or OS layers.
 
-Each buildpack layer blob MUST contain a single buildpack at the following file path:
+Each buildpack layer blob MUST contain a single [buildpack](https://github.com/buildpacks/spec/blob/main/buildpack.md) at the following file path:
 
 ```
 /cnb/buildpacks/<buildpack ID>/<buildpack version>/
 ```
 
-A buildpack ID, buildpack version, and at least one stack MUST be provided in the OCI image config as a Label.
+A buildpack MUST contain a `buildpack.toml` file at its root directory.
+
+#### Labels
+
+For each buildpack layer, the buildpack ID and the buildpack version MUST be provided in `io.buildpacks.buildpack.layers`
 
-Label: `io.buildpacks.buildpackage.metadata`
+The following labels MUST be set in the buildpack image(through the image config's `Labels` field):
+
+| Label             | Description | 
+| --------          | -------- 
+| `io.buildpacks.buildpack.metadata`     | A JSON object representing Buildpack Metadata   |
+| `io.builpacks.buildpack.layers`| A JSON object representing the buildpack layers |
+
+
+`io.buildpacks.buildpackage.metadata` (JSON)
 ```json
 {
   "id": "<entrypoint buildpack ID>",
+  "name": "<buildpack name>",
   "version": "<entrypoint buildpack version>",
-  "stacks": [
+  "homepage": "<buildpack home page",
+  "targets": [
     {
-      "id": "<stack ID>",
-      "mixins": ["<mixin name>"]
+      "os": "<build-image os>",
+      "architecture": "<architecture>"
```

Comment:
Not really sure, but should the target field also involve `variant`, `distribution` and `version`?